v1/translate.yaml

```diff
@@ -0,0 +1,254 @@
+info:
+  version: '1.0.0'
+  title: Content Translation API
+  description: Content Translation API
+  termsofservice: https://github.com/wikimedia/restbase
+  license:
+    name: Apache licence, v2
+    url: https://www.apache.org/licenses/LICENSE-2.0
+paths:
+  /machine/{from}/{to}{/provider}:
```

Comment:
Sounds good to me. Will move to `/transform`. @Nikerabbit @kartikm @santhoshtr any thoughts on having this under the domains' hierarchies? That is, instead of `https://wikimedia.org/api/rest_v1/transform/html/{src_lang}/to/{dest_lang}` to have `https://en.wikipedia.org/api/rest_v1/transform/html/to/{dest_lang}` ? Concretely, the question is: would it be acceptable to clients to assume that the domain against which the request is made is the source of the translation?